isc_dhcp_leases/iscdhcpleases.py

```diff
@@ -176,7 +176,11 @@ def __init__(self, ip, properties, options=None, sets=None):
         self.data = properties
         self.options = options
         self.sets = sets
-        _, self.binding_state = properties['binding'].split(' ', 1)
+        try:
+            _, self.binding_state = properties['binding'].split(' ', 1)
+        except KeyError:
+            print('Key error while trying to assign the binding property.\nPlease check if %s is available in your dhcp lease file.')
```

Comment:
I think you might have missed something in that warning. What should `%s` expand to?